client/src/document.js

```diff
@@ -99,12 +99,7 @@ function RenderSideBar({ document }) {
 }
 
 function SidebarLeaf({ depth, title, content }) {
-  const titleTag =
-    {
-      0: "h3",
-      1: "h4",
-      2: "h5"
-    }[depth] || "h6";
+  const titleTag = "h3";
   const titleNode = React.createElement(titleTag, null, title);
```

Comment:
If it's *always* a `h3` there's no need for the `React.createElement` trick and you can use JSX instead. 
```suggestion
  const titleNode = <h3>{title}</h3>
```